form_designer/tests/test_basics.py

```diff
@@ -24,14 +27,23 @@
 )
 
 
+class OverriddenDesignedForm(DesignedForm):
+    def clean_greeting(self):
+        return self.cleaned_data['greeting'].upper()
+
+
 @pytest.mark.django_db
 @pytest.mark.parametrize('push_messages', (False, True))
 @pytest.mark.parametrize('valid_data', (False, True))
 @pytest.mark.parametrize('method', ('GET', 'POST'))
 @pytest.mark.parametrize('anon', (False, True))
-def test_simple_form(rf, admin_user, greeting_form, push_messages, valid_data, method, anon):
+@pytest.mark.parametrize('override_form', (False, 'settings', 'kwarg'))
+def test_simple_form(
```

Comment:
This parametrization makes the testcase quite complex. I think keeping tests:
- simple (to avoid bugs in tests)
- easy to read and understand (tests act as kind of a specification)

is preferrable too.

No need to fix though. Just thinking out loud.